.travis.yml

```diff
@@ -17,3 +17,10 @@ install:
 script:
   - bash ./scripts/execute.sh
   - npm test
+deploy:
+  provider: npm
+  email: "patw.hi+github@gmail.com"
```

Comment:
Oh, sorry, it's not correct account.
I've defined this as `NPM_EMAIL`.

Reply:
ok updated that